nss-day-cohort-29/nutshell-midnight-dream
# Sort events

# Description
Events now display in chronological order. New events cannot be added without a name, date, and location.

## Type of change


# Testing Instructions
1. `git fetch --all`
1. `git checkout sort-events`
1. Run grunt
1. Click on "events" link
1. Check that all events display in chronological order with the next (or most impending) event showing up first
1. Click on "add event" button
1. Try to add an event without adding any input into the fields and ensure that an alert appears instructing you to fill in all fields before saving.

# Checklist:
